libs/sdk-ui-filters/src/AttributeFilter/AttributeDropdown/AttributeDropdownBody.tsx

```diff
@@ -1,13 +1,32 @@
 // (C) 2019 GoodData Corporation
 import React from "react";
-import { IAttributeElement } from "@gooddata/sdk-backend-spi";
 
 import { AttributeDropdownList } from "./AttributeDropdownList";
 import { AttributeDropdownButtons } from "./AttributeDropdownButtons";
-import { AttributeListItem } from "./types";
 import { AttributeDropdownItemsFilteredBody } from "./AttributeDropdownItemsFilteredBody";
+import { AttributeListItem } from "./types";
+import { IAttributeElement } from "@gooddata/sdk-backend-spi";
+import { ObjRef } from "@gooddata/sdk-model";
+import { IntlShape } from "react-intl";
 
-interface IAttributeDropdownBodyProps {
+/**
+ * @public
+ */
+export interface IAttributeDropdownListItemProps {
+    intl: IntlShape;
```

Comment:
Instead of
```
intl: IntlShape;
```

you can say 

```
export interface IAttributeDropdownListItemProps extends WrappedComponentProps
```

This is more in line to what the injectIntl does.